src/Platform/Microsoft.Testing.Platform.MSBuild/Tasks/TestingPlatformEntryPointTask.cs

```diff
@@ -135,11 +135,11 @@ internal sealed class TestingPlatformEntryPoint
 <System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage>
 Module TestingPlatformEntryPoint
 
-    Function Main(args As Global.System.String()) As Global.System.Int32
-        Return MainAsync(args).Result
+    Function Main(args As String()) As Integer
```

Comment:
Below (line 142) we use `Integer` (language keyword), while here we used `Global.System.Int32`. I'm switching to use language keywords every where.